### PR TITLE
fix TestLag.sol

### PR DESCRIPTION
**Pull request by jura-b**

### CODE REVIEW

File: test/sol/voracle/TestLag.sol [hunk 1]
```diff
@@ -13,24 +13,29 @@ contract TestLag {
     function beforeEach() public {
         mockMedianizer = new MockMedianizer(10);
         mockMedianizer2 = new MockMedianizer(11);
```
Comment (Kafakk) [inline test/sol/voracle/TestLag.sol]: remove

File: test/sol/voracle/TestLag.sol [hunk 2]
```diff
@@ -13,24 +13,29 @@ contract TestLag {
...
-        lag = new Lag(address(this), address(0));
+        lag = new Lag();
+        lag.initialize(address(this), address(0));
```
Comment (Kafakk) [inline test/sol/voracle/TestLag.sol:15]:
```suggestion
        lag.initialize(address(this), address(mockMedianizer));
```

Comment (jura-b) [reply test/sol/voracle/TestLag.sol:15]: we set mock medianizer on the case we really need

Comment (Kafakk) [reply test/sol/voracle/TestLag.sol:15]: okay

**Review by Kafakk — APPROVED** [review]

**Pull request merged**